nextflow_schema.json

```diff
@@ -95,41 +95,48 @@
             "fa_icon": "fas fa-magnifying-glass-chart",
             "description": "Options related to the downstream analyses performed by the pipeline.",
             "properties": {
-                "st_preprocess_min_counts": {
+                "st_qc_min_counts": {
                     "type": "integer",
                     "default": 500,
                     "description": "The minimum number of UMIs needed in a spot for that spot to pass the filtering.",
                     "fa_icon": "fas fa-hashtag"
                 },
-                "st_preprocess_min_genes": {
+                "st_qc_min_genes": {
                     "type": "integer",
                     "default": 250,
                     "description": "The minimum number of expressed genes in a spot needed for that spot to pass the filtering.",
                     "fa_icon": "fas fa-hashtag"
                 },
-                "st_preprocess_min_cells": {
+                "st_qc_min_spots": {
                     "type": "integer",
                     "default": 1,
                     "description": "The minimum number of spots in which a gene is expressed for that gene to pass the filtering.",
                     "fa_icon": "fas fa-hashtag"
                 },
-                "st_preprocess_hist_qc_max_total_counts": {
+                "st_qc_mito_threshold": {
```

Comment:
Any reason to have integer instead of float here?

Reply:
Nope! I changed it now.